packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -307,6 +325,12 @@ export class InsuredBridgeL1Client {
       this.instantRelays[l1Token] = {};
     }
 
+    // Fetch and store all rate model updated events, which will be used to fetch the rate model for a specific deposit
+    // quote timestamp.
+    this.updatedRateModelEventsForToken = this.updatedRateModelEventsForToken.concat(
+      await this._getAllRateModelEvents(blockSearchConfig)
+    );
```

Comment:
Do we need to sort these by block number, transaction index, and log index? See this code block where we do this elsewhere: https://github.com/UMAprotocol/protocol/blob/011905dcda88889884ec6f377d8b5cea8af20c7e/packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.ts#L231-L241.

I have generally seen them to at least be sorted by block number, but I haven't seen any documentation _guarantee_ that behavior.

Reply:
Yeah good point. I'm going to perform this sorting in `sdk/rateModel/createRateModelEventDictionary` but there isn't any need to sort them since the SDK function will sort them